pkgs/README.md

```diff
@@ -310,6 +310,7 @@ A (typically large) program with a distinct user interface, primarily used inter
 # Conventions
 
 ## Package naming
+[package-naming]: #package-naming
```

Comment:
I'm tending towards not having these, because otherwise people will think that they can use `#<left hand sid>` in URLs, when that's just accidental, and it would break when the header changes. So not having these doesn't give the illusion of stable anchors :)

```suggestion
```

Reply:
dropping the ones i added in this PR. We can prune the ones that already exist once an xrefcheck is in place